src/how-to-signer.md

```diff
@@ -1 +1,111 @@
 # How to operate a signer
+As a Stacker, you are responsible for ensuring the accuracy and integrity of incoming sBTC transactions. To satisfy this requirement and continue to earn BTC rewards, you must operate your own Signer or delegate your signing power to a third-party Signer. Below is a step-by-step guide to setting up and operating a Signer either by using the [default Signer implementation](#setting-up-a-default-signer) or by [utilizing the Signer SDK](#custom-signer-implementation).
+
+## Prerequisites
+Rust. To install, please [follow these instructions](https://www.rust-lang.org/tools/install).  
+Git
+
+## Setting up a Default Signer
+1. First, clone the Stacks sBTC mono repository:  
+```console
+git clone https://github.com/Trust-Machines/stacks-sbtc.git
+```
+2. Next, navigate to the stacks-signer-mini directory:  
+```console
+cd stacks-sbtc/stacks-signer-mini
+```
+3. Checkout the appropriate release branch you wish to use if you are not using the default main branch
+```console
+git checkout main
+```
+4. Compile the signer binary:  
+Note the binary path defaults to `target/release/signer`.
+```console
+cargo build --release
+```
```

Comment:
Nit: I'd recommend separating this into a "Installing the signer binary" step. Also I'd prefer it if the instructions used `cargo install --path stacks-signer-mini` to install the binaries, and subsequent commands would use `stacks-signer-mini ...` instead of  `cargo run --`